### ISSUE TITLE
deploy to heroku without troubless

### ISSUE THREAD

**Issue by atherdon** [GroceriStar/fake-api#25]

**Comment by vadim9999** [comment]
where can I find 100% working code on heroku?

**Comment by atherdon** [comment]
you mean fake-api deployed at heroku?
or you need access to settings at heroku account?
URL saved in description of this repository

**Comment by atherdon** [comment]
I can just add you as a collaborator. or i'm ok that you will try on your
own. i think there is not something "big" happens. I think the latest
version of express is just required some additional tuning. and it's not
about our server setup or our heroku configuration

**Comment by atherdon** [comment]
it's also not pressing issue for now. i mean if you can run server locally - it can wait

**Comment by vadim9999** [comment]
I fixed it

**Comment by vadim9999** [comment]
You can see it in my deploy on heroku https://rocky-island-20527.herokuapp.com/hello

**Comment by atherdon** [comment]
i see it on PR build page ;)
looks cool.
merging. can you also apply the same changes at graphql-server?

**Comment by atherdon** [comment]
btw, did you add any custom settings to heroku? i mean i deploy it
automatically from github repo(lazy variant)

**Comment by vadim9999** [comment]
I doesn't change any parameters in heroku It's problem was in server port and ip

**Comment by vadim9999** [comment]
Do you mean deploy graphql to heroku?

**Comment by atherdon** [comment]
no, i mean you made some changes, and tests start to work. i assume code in
the last 3 commits that you've made
at tests we calling an url and it return a basic value - telling us that
server is not dead.
and at graphql we have a same expressjs setup, so it wouldn't be a problem
for you.

**Comment by vadim9999** [comment]
Ok, I will do it
